### ISSUE TITLE
Release to production Pypi registry https://pypi.org/

### ISSUE THREAD

**Issue by tombruijn** [appsignal/appsignal-python#29]

**Comment by unflxw** [comment]
> Test with https://github.com/appsignal/mono/pull/57
I did test it on the actual repo locally -- still good for someone else to take a second look, but it should work.

**Comment by unflxw** [comment]
This is blocked as PyPI will not allow packages with the name "appsignal" to be released. An issue has been opened in the corresponding PyPI repo: https://github.com/pypi/support/issues/2817

Slack conversation: https://appsignal.slack.com/archives/C055A2H9NPJ/p1683554468022909

**Comment by unflxw** [comment]
Agreed. We could always do what the `bundler` gem does for [`gem install bundle`](https://rubygems.org/gems/bundle/) (without the `r`) -- that is, turn `appsignal-beta` into a package whose only dependency is `appsignal`.

**Comment by tombruijn** [comment]
As discussed let's try and release a package with the name like `appsignal-test` or `appsignal-beta` to get _a package_ out the door to test.

**Comment by tombruijn** [comment]
Moving to blocked. Everything has been done, except publishing with the desired package name "appsignal". Now available as "appsignal-beta".

**Comment by tombruijn** [comment]
@unflxw will ask for help on IRC.
